src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/EntityAIWorkLumberjack.java

```diff
@@ -164,6 +179,11 @@
      */
     private PathResult pathToTree;
 
+    /**
+     * A list of all planted nether trees
+     */
+    private final List<BlockPos> netherTrees = new ArrayList<>();
```

Comment:
I think you want to move the whole nether tree list and possibly the bonemealing into the lumberjack building itself, as its to simulate natural growth which takes place anytime and not just when the AI is active, and you can save the positions list there so it persists during reloads

Reply:
could even use the slow world tick in the lumberjack building for this.

Reply:
ye thats what I was thinking

Reply:
also make it a Set<BlockPos> then you dont need to worry about duplicated positions